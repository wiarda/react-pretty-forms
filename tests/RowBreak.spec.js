/* global describe test expect */
import React from 'react';
import renderer from 'react-test-renderer';
import { RowBreak } from '../index';

describe('TextArea component tests', () => {
  test('Renders consistently', () => {
    const component = renderer
      .create(<RowBreak styles={{ test: 'margin: auto' }}>This is a child element</RowBreak>)
      .toJSON();
    expect.assertions(1);
    expect(component).toMatchSnapshot();
  });
});
