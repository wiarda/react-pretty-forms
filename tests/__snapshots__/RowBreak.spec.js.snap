// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea component tests Renders consistently 1`] = `
<div
  className="pretty-row"
>
  This is a child element
</div>
`;
